.github/workflows/build.yml

```
@@ -0,0 +1,32 @@
+name: Build
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.23.1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Build and push container image
+        run: |
+          export REGISTRY=gchr.io/mesosphere
+          export VERSION=v2.7.0-ntnx.1
```

```suggestion
          export VERSION=${{ github.ref_name }}
```

This will be reusable for any tag we'll push.